test/MUIDataTableBody.test.js

```diff
@@ -493,4 +493,31 @@ describe('<TableBody />', function() {
 
     expect(html).to.contain('Test_Text');
   });
+
+  it('should selectedRows be passed in isRowSelectable', () => {;
```

Comment:
The wording of this threw me off a little bit. Can we change it to "should pass in selectedRows to isRowSelectable"?